hypha/apply/funds/views.py

```diff
@@ -1579,6 +1583,45 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class SubmissionWithdrawView(SingleObjectTemplateResponseMixin, BaseDetailView):
```

Comment:
Add

`@method_decorator(login_required, name="dispatch")`

Reply:
Is the `SingleObjectTemplateResponseMixin` needed?

Reply:
:shrug: I suppose I could try without once the other issues are resolved.